### PR TITLE
Update SpringMVC model to Springfox 2.4.0

### PR DESCRIPTION
**Pull request by diyfr**
Update Pom &  swaggerConfig to Springfox 2.4.0

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache [hunk 1]
```diff
@@ -121,10 +121,10 @@
     <swagger-core-version>1.5.8</swagger-core-version>
     <jetty-version>9.2.9.v20150224</jetty-version>
     <jersey-version>1.13</jersey-version>
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache]: @diyfr  do you mind updating jersey version to 1.19.1  (the latest version)?

File: modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache [hunk 2]
```diff
@@ -121,10 +121,10 @@
     <swagger-core-version>1.5.8</swagger-core-version>
     <jetty-version>9.2.9.v20150224</jetty-version>
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache]: For jetty, would you please upgrade it to use 9.2.15.v20160210 (the latest stable version for java7)?

**Comment by wing328** [comment]
@diyfr thanks for the PR. Please run the following commands to update the springmvc petstore sample, which will be verified by the CI:
```
./bin/spring-mvc-petstore-j8-async-server.sh
./bin/spring-mvc-petstore-server.sh
```
Also left you 2 comments about upgrading other dependencies.

File: modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache (hunk 1 above)
Comment (diyfr) [inline modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache]: I do not find what this dependency property is used . Never used ? Delete ?
http://mvnrepository.com/artifact/io.swagger/swagger-jersey-jaxrs/1.5.8

File: modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache (hunk 2 above)
Comment (diyfr) [inline modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache]: I prepare a version of springboot so I do not use jetty and this was the version available in enterprise maven repository.

**Comment by diyfr** [comment]
Why I get errors on javascript while I did not commits on this?

**Comment by wing328** [comment]
@diyfr thank you for contributing the batch files (windows) for generating the petstore springmvc samples

**Comment by wing328** [comment]
@diyfr may I know if you also have time to update https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom-j8-async.mustache#L169, which is the pom for Java 8?

**Pull request merged**